tensorboard/webapp/metrics/views/card_renderer/scalar_card_component.ts

```diff
@@ -258,4 +262,10 @@ export class ScalarCardComponent {
         });
     }
   }
+
+  openDataDownloadDialog(): void {
+    this.dialog.open(DataDownloadDialogContainer, {
+      data: {cardId: this.cardId},
```

Comment:
Can you test that you open the correct dialog? I don't think I saw one in data_download_dialog_test.ts. I would expect one in scalar_card_test.ts.